apps/i18n/common/en_us.json

```diff
@@ -2223,6 +2223,7 @@
   "startOver": "Start Over",
   "startOverTitle": "Are you sure you want to start over?",
   "startOverBody": "This will reset the puzzle to its start state and reset all the data you've added or changed.",
+  "startOverWorkspace": "This will reset the workspace to its start state and remove all the blocks/code you've added or changed.",
```

Comment:
I would simplify to just one of "blocks" or "code".   Using "blocks" would be consistent with existing tutorials.  Also should we capitalize "Workspace" to match the heading?

Reply:
Yeah, I was trying to find a generic error message since this dialog would be used for multiple labs, but agreed that just "blocks" is clearer and more consistent. I can use "blocks" for now, and later we can try to configure this message based on the lab?